_pages/ato/checklist.md

```diff
@@ -31,13 +31,16 @@ You are welcome to ask any questions as comments in the issue or #devops.
 
 ### Project team
 
+1. [Set up monitoring](https://pages.18f.gov/before-you-ship/infrastructure/monitoring/)
+    * [ ] [Downtime alerts](https://pages.18f.gov/before-you-ship/infrastructure/monitoring/#downtime)
+    * [ ] [Error alerts](https://pages.18f.gov/before-you-ship/infrastructure/monitoring/#errors)
 1. [ ] Add an [`about.yml`](https://github.com/18F/about_yml) for the main repository
-1. [ ] Run the [security scans](https://pages.18f.gov/before-you-ship/security/scanning/)
-    * [ ] [Static analysis](https://pages.18f.gov/before-you-ship/security/static-analysis/)
-    * [ ] [Dynamic scans](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
-    * [ ] Add badges to the README
-1. [ ] Resolve any visible security issues, re-running the scans as needed
-1. [ ] Add the issue-free scan reports to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing)
+1. [ ] [Security scans](https://pages.18f.gov/before-you-ship/security/scanning/)
+    * [ ] Set up [static analysis](https://pages.18f.gov/before-you-ship/security/static-analysis/) service
+        * [ ] Add service badges to the README
+    * [ ] [Run OWASP ZAP](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
+        1. [ ] Resolve any visible security issues, re-running the scan as needed
+        1. [ ] Add the issue-free scan report to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing)
```

Comment:
Updates inspired by @harrisj's additions to the checklist in https://github.com/18F/DevOps/issues/544.